aerial_robot_nerve/spinal/mcu_project/TrueSTUDIO/spinal/spinal_converter.log

```diff
@@ -0,0 +1,33 @@
+Project: spinal
+Converter: Atollic TrueSTUDIO project converter
+Date: 20210414
```

Comment:
I think we can add this log file into .gitignore.

Reply:
I know, I just wanted this log to be included during the migration. I'll add this log (and also debug launch file, maybe) into .gitignore.